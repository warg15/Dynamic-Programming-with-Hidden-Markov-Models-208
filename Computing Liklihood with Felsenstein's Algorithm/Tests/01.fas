>Pigeon
AAGAATGCAGGTCTCCGTCTACACTCGTTGCCCCTTCTTTGAAGGTGCCAATACAGGAGACTCCGTCGCTCTACTTGTCTGTACTACACTTATTCCTCGTACTCCGTGGGCAAAAATGAGCACAGGGCATATCTCCTTGCCGCTGGCATCGTTAATCAGCATAGGGTGACTCCCGTCCGACTCAAGAAACGAGCTAAGCG
>White-tailed-eagle
ACGGACGCAGGTCTTCGTCTCTTGTTGTTGCCCCTTCTTTGAAAATGAAAATGCAGGGGATTCCGTCGCTCCACTTGTCCATATTACAATTATTGGTCGTACCCCGTGGCCCAAAATGAGCACAGGGCGCATCTCCTTGCCGCCGGCATCTTTAATCGGCATAGGGGTAATCGTGGCCGACTCAAGAGACGAGTTAAGCG
>Speckled-mousebird
AAGAACGCAGAGCTTCGTCTCTTCTTTTTGCCCCTGTTGTGAAAATGACAATACATGGGACTGCGTCGCTCTACTCGTCCATACTAGAATTGTAGCTCGTACTACGTGGCCAAAAATGAACATAGGGCGCGGCTCTTTGCCGCCAGCATCTTTAATCGACATAGGGGGAATCGTGGCCCACTTAAGAAACGAGTTAAGCG
>Hoatzin
AAGGGCGCAGGTTTTCGTCTATTCTTATTGCCCCTTCTTTGAAAATGATAATGCCGGGGACTCCGTCGCTCTACTTGTCCATACTGAAATTATCCCTCGTACTCCGTGGCCAGAAATGAGCACAGGGCGCATCTCGTTGCCGCCGGCATCTTTAATCGGCGTAGAGGGAATTGTAGCCGACGCAAGAAACGAGTTAAGCG
>Rifleman
AAGGGAGCAGGACTTCGTCTCTTCTCGTTGCCCCTTCTTTGAAAATGACTCTGCAGGGGACTCTATCGCTCTGCTTGTCCATACTATAATTATTCCTCGTACTCCGTGGCTAAAAATGAGCACAGGGCGCATCTCCCTGACGTCGGCATCTTTAGTCGGCTTATGGGCAATCATGGCCAACTCAAGAAGCGAGTGAGGCC
>Great-crested-grebe
AAGAACGCAGGTCTTCGGCTATTCTTGTTGCCCCTTCTTTGAAAATGACAATGCAGGGGGCTCCGTCGCTCTCCTTGTCCATACTACAATTATTCCGCGTATTCAGTGGCCAAAGATGAACAGAGGGCGCATCTCCTTGTCGCCAGCAGCTTTCGTGGGTATAGGGGGAATCGTGGCCGACTCAAGAAACGAGTTAAGCG
>Brown-mesite
AAGAACGCAGGTCTACGTCTATTCTTGTTGGCCCTTCTTTGAAAATGACAGTGCAGGGGACTCCGTCGCGCTACTTGTCCATACTACAATTATTTCTCGTACTCCATGGCCAAGAATGAGCACAGGGTGTATCTCCTTGCCGCCGACATCTTTAATCGGCATAGGAGGAATCGTAGCCGGCTCAAAAAACGAGTTAAGCG
>Emperor-penguin
AAGGACGCAGGTCTTCGTCTCTTCTTGTTGCCCCTTCCTTGAAAATGACAATGCAGGGGACTCCGTCGCTCTACGTGTCCATACTACAGTTATTCCTCGTACTCCGTGGCCAAAAATGAGCACAGGGCGCATCTCCTTGCCGCCGGCATCCTTAATCGGCATAGGGGGATTCGTGGCCGACTCAAGAAACGAGTTAAGCG
>Peregrine-falcon
AAGGACGCAGATCTTCGTCTCTTCTTGTTGCCCCTTCTTAGAAAAAGACAATGCAGGGGACTCCGTCGCTCTACTTGTCTATACTACAAGTATTCCTCGTACTTCGTGGCCAAAAATGAGCACAGGGCACATCTCCTTGCCGCTGACACCTTTAATTGGCATAAGGGGAATCATGGCCGACTCAAGAAACGAGTTAAGCG
>Dalmatian-pelican
AAGGACGCAGGTATCCGTCTCTTCTTGTTTCCCCTTCTTTGAAAATGACAATGCAGGGGACTCCGTCGCTCTACTTGTCCACACTATAATTATTCCTCGTACCCCGTGGCCAAAAATGAGCACAGGGCGCATCTTCTTGCCGCCGGCAGCTTTAATCGACATAGACGGAATCGTGGCCGACTCAAGAAACGAACTAAGCG
>Turkey
AAGAGCGCAGGTCCTTGTCTATTCTAGCTGCCCCTTTTTTGAAAATGACAATGCGAGGCGCTCCGCCTCTCTATTTGCTTATACTACAATTATTCATCGTACTTCGTGGCCAAAACTGAACGCTGGACGCACCTTCTTGCCGCCGGCATATTTAATCATCATAGGGGGTATAGGGGCCGACTCAAGAAACGGGTCAAGCA
>Carmine-bee-eater
AAGGAGGCAGCTCTTCGTCTCGTCTCGTTCCCCCTTCTTTAAAAATGATAATGCAGGGGACTCCGTCGCTCTATTTGTCCATACTACCAGTATTCTTCGTACTCAGTGGCCAAAAATGAGCACAGGGCGCATCTCCTTGCCGCAAGCATCTTAAATCGGCTCAGGGGGGATCGTGGCCGGCTCAAGAGCAGAGTTAAGAA
>Rhinoceros-hornbill
AAAGACGCGGGTCTTCATCTCTTCTTGTCGCCCCTTCCTTGAAAATGACAATGCAAGGGACTCCATCGCTCTACTTGTCCATACTATAATTATTCCTCGTACTCCGTGGCCAAGGATGAGCACGGAGCGCATCTGGTAGCCGCCGGCATCTTTAATCGGCATAGGGGGAAACGGGGTCAACCCAAGAAACGAGTTAAGTG
>American-crow
AAGAAACCAGGACTTCGTCCCTTCCTGTTGTCCCTTCTTTGAAAATGACTCTGAAGGGGTCTCGATCGCTCTACCTGTCCATCCTATAATTATTAATCATACTCCGTGGCCGAAAATCAGGACAGGGCGCATCTCCCTGCCGCCGGTATCTCTAAGCGGCATAGGGGGAATCGCGGCCGACTCAAGAAACGAGTTAAGCG
>Cuckoo-roller
AAGGACGCAGGTCTTCGTCTCTTATTGTTGCCCCTCCTTTTAAAACGACAATCCAGGGGACTCCGCCGTTCTATTTATCCATACTACAATTATTCCTCGTACTCCGCGGCCAAAAATGAGCACAGGGCGTATCTCCTTGCCGCTGGCATCTTTAATCGGCATAGAGGGAATCGTGGCCGACGCAAGAAACGAGTTAAGCG
>Chuck-wills-widow
AAGGACGCAGGTCTTCCGCTATTCTTGTTGCCCCTTCTTTGAAAATGACAATGCAGGGGACTCCGTCGCTCTACTTGTCCATACTACAATTAATCCTCGTACTCCGTGGCCAAAAACGAGCACAGGGCGCATCTCCTTGCCGCTGGTATCTTTAATCGGTATAGGGGGAATCGTGGCCGACTTAAGAAACGAGTTAATCG
>Bald-eagle
ACGGACGCAGGTCTTCGTCTCTTGTTGTTGCCCCTTCTTTGAAAATGAAAATGCAGGGGATTCCGTCGCTCCACTTGTCCATATTACAATTATTCGTCGTACCCCGTGGCCCAAAATGAGCACAGGGCGCATCTCCTTGCCGCCGGCATCTTTAATCGGCATAGGGGTAATCGTGGCCGACTCAAGAGACGAGTTAAGCG
>Common-ostrich
AAGAACACAGGCGTTCGTCTATTCCTGTTGCCCGTGCTTTGCAAATAACAATGGAAGGGACTCCGCCGTTGTACTTGTCCACACTGCAATTGTTTCTCGTACTCCGTGGCTAAACATGAGCACAGGGCGCATCTCATTACCGCTGTTGTATTCAGTCGGGATAGGGGGAATCGTGGCCGACTCAATAGACAAATTAAGCG
>Chicken
AAGAACGCAGCTTCTTGTCTATTCTAGTTGCCCCTTCTCTGAAAATGACAATGCAAGGAACTCCGCCTCGCCATTTGCCTATACTACAATTATTCATCGTACTCCGTGGCCAAAACTGAACGCAGGGCGCATCTTCTTCCCGCCGGCATATTTAATCATCATAGGGGGTATCGTGGCCGACTCAAGAAACGGGTCAAGCG
>Crested-ibis
AAGGACGCAGGTCTTCGTCTCTTCTTGTTGCCCCTTCTTTGAAAATGACAATGCAGGGGACTCCGTCGCTCTACTTGCCCATACTACAATTATTCCTCGTACTCCGTGGCCAAAAATGAGCACAGGGCGCATCTCCTTGCCGCCGGCATCGTTAATCGGAATAGGGGGAATCGTGGCCGACTCAAGAAACGAGTTAAGCG
>Medium-ground-finch
ATGGAACTAGGACTTCGTCGCTTCCTGTTGTACCTTCTTTGAAAATGACTCTGAAGGGGTCTCTATCGCTCTACCTGTTCATACTATAATTATTAATCGTACTCCGTGGCCGAAAGTCAGAACGAGGCGCATCTCCCTGCCGCCGGTATCTCCAATCGGCATACGGGGAATCGTGGCCGACTCAAGAAGCGAGATCAGCG
>Red-crested-turaco
GAGAACGTGGGTCTTCGTATATTCTTGTTGCCCCTTCTGTGAAAATGACAATGCGAGGGACTCCATCGCTCTACTTGTCCATACTACAATTATTCCTCGTCCTCTGTGGCCAAAAATGAGCACAGGGTGCGTCTCCTTACCGCCGGTACCTTCAATCGGCATAGGGGGAATCGGGGCCGACTTAAGAAACGAGTTAGGGG
>Common-cuckoo
AAGGACGCAGGTCTTCGTCTATTCTTGTTGCCCTTTCCTTGAAAATGACAGTGCAGGGGACTTCGTCGCTCTACTTGTCCACGTTACAATTATTGCTCGTACTCCGTGGCCAAAAATCAGCACAGGGCGAATCTCCTTGCCGCCGGCATCTCCAATCGGCATAGGGGGAATTGTGGCCGACTTAAGAAACGAGTTAAGCG
>Killdeer
AAAGACGCAGGTCCTCGTCTATTCTTGTTGCCGTTTCTTTGAAAATGACAATGCAGGGGACTCCGTCGCTCTACTTGTCCATACTACAATTATTCCCCGCACTCCTTGGCCAAAAATGAGCACAGGGCGCGTCTCCTCGCTGCAGGCACCTGTAATCGGCGTAGGGGGAATCGTGGCCGACTCAAGAAACGAGTTAAGCG
>American-flamingo
AAGAACGCAGGTATTCGTCTATTCTTGTTGCCCCTTCTTTGAAAATGACAATGCAGGGGACTCCGTCGCTCTACTTGTCCATGCTACAATTATTCCGCGTACTCAGTGGCCAAAAACGAACACAGTGCGCATCTCCTTGTCGCCGGCATCTTTCGTGGGCATAGGGGGAATCGTGGCCGACTCAAGAAACGAGTTAAGCG
>Chimney-swift
GAGGACGCAGGTCTTCGTCTATTCTTGCTGCCCCTCCGTTGAGAATGACAGTGCAGGGGATTCCGTCGTTCTGCTTAGCGATATTTCAATTATTCCTCGTACTATGCGGCCACAAAGGAGCACAGGGCTCATCTCCTTGCTACCGGCATCCGTACGCGGCATAGGGGGAATCATGGTCGATTTAAGAAACGAGTCGGGCG
>Red-legged-seriema
AAGGACGCAGGTCTTCGTCTACTCTTGTTGCCACTTCTTTGAAAATGACAATGCAGGGAACTTCATCGCTCTACCTATCCGTACTACAATTATTCCTCGTACTCCGTGGCCAAAAATGAGCACAGGGCGCATCTCATTGCCATCGGCATCTCTAATCGGCATAGGAGGAATCGTGACCGACTCAAGAAACGAGTTAATCG
>Adelie-penguin
AAGGACGCAGGTCTTCGTCTCTTCTTATTGCCCCTTCCTTGAAAATGACAATGCGGGGGACTCCGTCGTTCTACTTGTCCATACTACAGTTATTCCTCGTACTCCGTGGCCAAAAATGAGCACAGGGCGCATCTCCCTGCCGCCGGCATCTTCAATCGGCATAGGGGGAATCGTGGCCGACTCAAGAAACGAGTTAAGCG
>White-throated-tinamou
AAGAATGCAGGCGTTCGTCTATTCCTGTTGCTCCCTCTTTGGAAGTGCCAATGCTGGAGACTCGACCGTTCTACTTGTTCATACTGCGATCATTGCTCGTACTCCGTGGCTAGAGATGAGGATAGGCCGCATCTTATTGCCACCAATGTATTTAATCGGAATAGGAGAAATCGTGGCCTATTTGATAAACGAGTTAAGCG
>Golden-collared-manakin
TAGGAAGCGGGGCTTCGTCCCTTCTTGTTGTCCCTTCTTTGAAAATGACTATGCAGGGGACTCTATCGCTCTACTTGTCCATACTATAGTTATTCCTCGTGTCCCGTGGCCGAAAATGAGCACAGGGCGCTTCTACCTGCCGCCGGCATCTTTAATCGGCATAGGGGGAATCGTGGCCGACTCAAGAAACGAGTTGAACG
>Little-egret
AAGGACGCAGGTCTTCGTCTCTTCTCGTTGCCCCTTCTTTTACAATGACAATACAGGGGACTCCGTCGCTCTACTTGTCCGTACTACAATTATGCCTCGTACTCCGTGGCCAACAATGAGCACATGGCGTATCTCCTTACGGCCGGCATCTTTAATCGACATAGGCGGAATCATGGCCGACTCAAGAAACGAGTTAAGCG
>Pekin-duck
AAGGACACTGGTCTTGGTCTATTCTCGTTGCCCCTTCTCTGAAAATGACAATGCAGGGGACTCCGTCGCTCTACTTTTCGATGCTACAATTATTCCTCGTATTCCGGGGCGAAAACTAAGCGCAGGGCAGATCTCCTTACCGCCGGCATATATCATCGTTATACGGGGGATCGCGGCCGACCCAGGAAACGAGTTAAGTG
>Grey-crowned-crane
AAAGACCCAGCTCATCATCTATTCTCGTTGCCCCTTCTTTGAAATTGACAATGCAGGGGACTCCGCCGCTCTACTTGTCCATACTACAATTATTCCTCGTACTCCGTGGCCAAAACTGAGCACAGGGCACATCTCCTTGCCGCCGGCATCTTTAATCGGCATAGAGTCAATCATGGTCGACTCAAGAAACGAGTTAAGTG
>Bar-tailed-trogon
AAGGACGTAGGCCTTCGTCTCCTCTTGCTGCCCCTTCGTTGAAATTTACAACGCACGGGGCTCCGGTGCTCTAGTTGTGTATACCATAATTATTCCTCGTACTTCGTAGCCAAAAATGAGCACAGGGCGCCTCCCCTTGTCGTCGGTATCTTTAAGCGGCATAAAGGGAATCGTGGCCGACGCAAGAAACGAGTTAAGCG
>Yellow-throated-sandgrouse
AAGAACGCAGGTCTTCGTCTAATCTTGTTGCCCCTTCTTTGAAAATCACAGTGCAGGGGACTCCGTCGCTCTACTTGTCCATACTATAATTATTCCTCATACTTCGCGGCCGAAAATGAGCACAGGTCGCATGTCCTTGGCGCGGGCATGTTTAATTAGCATAGGGGGAATTGTGGCCGACTCAAGAAACGAGTTAAGCG
>Kea
AAGTACGCAGGTCTTCATCTATTCTTGTTGCGCCTTCTATGAAAATGACAATGCAGGAGACTCCGTCTCTCTACTTGTCCATACTACAATTATTCCTCGTACTCCATGGCCAAAAATGAGCACAGGGCGCATCGCCTTGCCGCCGGCATCTTTGATCGGCATAGGGCGAATTATGGCCGACTCAAGAAACGAGTTAAGCG
>White-tailed-tropicbird
AAGGACGGAGGTCTTCGTCTCTTCTTGTTGCCCCTTCTTTGGAAATGACAATGCAGGGGACTCCGTCGCTCTACTTGTCCATACTACAATTATTCCTCATACTTCGTGGCCAAAAATGAGCACAGGCGGCATCTCCTTTCCGCCGGCATCTCTAACCGGCATAGGGGGAATCGTGGCCGACTCAAGAAACGAGTTAAGCG
>Northern-fulmar
AAGGACGCAGGTTTTCGTCTCTTCTTGTTGCCCCTTCTTTGAAAATGACAATACAGGGGACTCCGTCGCTCTACTTGTCCATACTACAATTATTCCTCGTACTCCGTCGCCCAAAATGTGCACAGGGCGCATCTCCTTGCCGCCGGCATGTTTAATCGGCATAGGGGGAATCGTGGCCTACTCAAGAAACGAGTTACGCG
>Annas-hummingbird
GAGGCCGCAGGTCTTCGTTCATTCTTGTTGCCCTTATTTCGAAAATGACAATGGTGGGGACTCCATTGCTCTACTTGTCCATACTTCAATTATTCCTAGTCCTCCGTGACCAAAACGAAGCACGGGGCTTATCTCCTTGCTGCCGGCATCTTTACACGGCATAGGGGGAATCGTGGCCGAGATAAAAAACGAGTTAGGTA
>Red-throated-loon
AAGGAGGCAGGTCTTCGTCTCTTCTTGTTGCCCCTTATTCGAAAATGACAATGCAGGGGACTCCGTCGCTCTGCTTGTCCATACTACAATTATTCCTCGTACTCCGTAGCCAAAAATGAGCACAGGGCGCATCTCCTTACCGCCGGCATCTTTAATCGGCATAGGGGGAATCGTGGCCGACTCAAGAAACGAGTTAAACG
>Turkey-vulture
AAGGACGCAGGTCTTCGTCTCTTCTTGTTGCCCCTTCTTTGAAAGTGACAACGCAGGGGTCTACGTCGCTCTACTTGTCCATACTACAATTATTCCTCGTACTCCGTGGCCAAAAATGAGTACAGGTCGCATCTCCTTGCCGCCGGCATCTTTGATCGGCATAGTGGGAATCGTGGCCGACTCAAGAAACGAGTTAAGCG
>MacQueens-bustard
AAGGACGTAGCTCTTCGTCTATTCTTGTTGCCCCTTCCTTGAAAATGACTATGCAGGGGATTCCGTCACTCTACTTATCCATACTACAATTATTCCTCGTACTCCGTGGCCAAAAAAGATCACAGGGCGCATCTCCTTGCCGCCGGCATCTTTAATCGGCATAGAGGGAATCGTGGCCGACTTAAGAAACTGGTTAAGCG
>Great-cormorant
AAGGACCCAGGTCTTCGTCTCTTCTTTTTGCCCCTTCTTTTGAAATGACAATGCAGGGGACCCCGTCGCTCTCCTTGTTCATATTTGAATTATTCTTCGTACTCCGTGGCCAAAAATGAGCACAGGGCGCATTTCCTTGCCGCCGGCAGCTTTAATCGGCATAGGTGGAATCGTGGCCAACTCAAGAAACGAGTTAAGCG
>Budgerigar
AAATACGCAGGTCTTCATCTCTTCTTGTTGCCTCTTCTATGAAAGTGACAATACAGGAGGCTCCGTCTCTCTATTTGTCCATACTACAACTACTCCTCGTACTCCACGACCAAAAATGAGCACAGAGCGCATCGCCTTGCCGCCGGTATCTTTAGTCGGCATAGGGCGAATCGTGGCCGACTCAAGAAACTAGTTAAGCG
>Sunbittern
AAGGACGCAGGTCTTCGTCTCTTCTTGTTGCCGCTTCTTTGAAAATGACAATGCAGGGGACTCCGTCGCTCTACTTGTCCATACTACAATTATTCCTCGGACTCCGTGGCCAGAAATGAGCACAGGGCGTATCTGCTTGCCGTCGGCATCTTTAACCGGCATAGGGGGTATCGCGGCCGACTCGAGAAACCAGTTAAGCA
>Zebra-finch
AAGGAACCAGGACTTCGTCCCTTCCTGTTGCCCCTTCTTTGAAAACGACTCTGAAGGGGTCGCTATCGCTCTACCTGTTCATACTGTAATTGTTAATCGTATTCCGTGGCCAAAAGTCAGAACAGGGCGCATCTCCCTGCCGCAGGTAGCTCTAATCGGCATAGGGGGAATCGTGGCTGACTCCAGAAACGAGATCAGCG
>Barn-owl
AAGGACGCAGGTCCTCGTCCCTTCTCGTTGCCCCTTCTTTGGAAATGACAATGCAGGCGAATCGGTCGCTCTACTCGTCCATACTATAATTATTCCTGGTACTCCGTGGCCAAAAATGAGCACAGGGCGAATCTCGTCGCGGCTGGCATCTTTAACCGACATAGGGGGGATGGTGGCCGACTCAAGAAACGAGTTAAGTG
>Downy-woodpecker
AAGGACGCAGATTTTGATCTCTTCTTGTTACCACTTCTTTGAAAATGACAATGCAGGGGACCCCACCGCTCTACTCGTCCATACTACAATAATTCCTCGTACTCCGTAGGCAAAAATCAACACAGGACACATTTCTTTGCCGCAGGCATCTTTAATAGGCATAAGGGGAATCATGGCCGACTCAAGGACCGAGTTAAACG
