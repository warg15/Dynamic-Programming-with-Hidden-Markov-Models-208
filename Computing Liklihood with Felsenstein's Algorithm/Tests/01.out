-3718.4816604205585
